app/views/pages/faqs.html.erb

```diff
@@ -6,6 +6,270 @@
   <div class="usa-width-three-fourths usa-layout-docs-main_content">
     <h1><%= @page_title %></h1>
 
-    <%# TODO Add page content %>
+    <%# TODO Hide the checkboxes when the screen is too narrow %>
+    <div class="usa-grid" id="faq-filters">
+
+      <div class="usa-width-one-third">
+        <fieldset class="usa-fieldset-inputs usa-sans">
+          <ul class="usa-unstyled-list">
+            <li>
+              <input id="claims-checkbox" type="checkbox" value="claims">
+              <label for="claims">Claims</label>
+            </li>
+            <li>
+              <input id="entitlements-checkbox" type="checkbox" value="entitlements">
+              <label for="entitlements">Entitlements</label>
+            </li>
+            <li>
+              <input id="hhg-checkbox" type="checkbox" value="hhg">
+              <label for="hhg">Household Goods (HHG)</label>
+            </li>
+          </ul>
+        </fieldset>
+      </div>
+
+      <div class="usa-width-one-third">
+        <fieldset class="usa-fieldset-inputs usa-sans">
+          <ul class="usa-unstyled-list">
+            <li>
+              <input id="nts-checkbox" type="checkbox" value="nts">
+              <label for="nts">Long-term Storage (NTS)</label>
+            </li>
+            <li>
+              <input id="tsp-checkbox" type="checkbox" value="tsp">
+              <label for="tsp">Movers (TSPs)</label>
+            </li>
+            <li>
+              <input id="moving-day-checkbox" type="checkbox" value="moving-day">
+              <label for="moving-day">Moving Day</label>
+            </li>
+          </ul>
+        </fieldset>
+      </div>
+
+      <div class="usa-width-one-third">
+        <fieldset class="usa-fieldset-inputs usa-sans">
+          <ul class="usa-unstyled-list">
+            <li>
+              <input id="scheduling-checkbox" type="checkbox" value="scheduling">
+              <label for="scheduling">Scheduling</label>
+            </li>
+            <li>
+              <input id="delivery-checkbox" type="checkbox" value="delivery">
+              <label for="delivery">Shipment Delivery</label>
+            </li>
+            <li>
+              <input id="techhelp-checkbox" type="checkbox" value="techhelp">
+              <label for="techhelp">Technical Help</label>
+            </li>
+          </ul>
+        </fieldset>
+      </div>
+
+    </div>
+
+    <ul class="usa-accordion-bordered" id="faq-accordions">
+
+      <li class="faq-accordion" data-categories="claims">
+        <button class="usa-accordion-button faq-accordion-button" aria-controls="faq-claim-limit-content">
+          How long do I have to file a claim?
+        </button>
+        <div id="faq-claim-limit-content" class="usa-accordion-content">
+        </div>
```

Comment:
Suggest adding `<%# TODO Add FAQ content %>` in these empty `<div>`s so they show up when running `bin/rails notes`.